sql-plugin/src/main/scala/com/nvidia/spark/rapids/SparkShims.scala

```diff
@@ -62,6 +65,11 @@ case class SparkShimVersion(major: Int, minor: Int, patch: Int) extends ShimVers
   override def toString(): String = s"$major.$minor.$patch"
 }
 
+case class ClouderaShimVersion(major: Int, minor: Int, patch: Int, clouderaVersion: String)
```

Comment:
there are a few places we use these Versions that should be checked if they need updated:
https://github.com/NVIDIA/spark-rapids/blob/branch-21.06/tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala#L1839
https://github.com/NVIDIA/spark-rapids/blob/branch-21.06/tests/src/test/scala/com/nvidia/spark/rapids/StringFallbackSuite.scala#L36